src/storage/kv/perf_context.rs

```diff
@@ -31,6 +31,7 @@ impl slog::KV for PerfStatisticsInstant {
 impl PerfStatisticsInstant {
     /// Create an instance which stores instant statistics values, retrieved at creation.
     pub fn new() -> Self {
+        set_perf_level(PerfLevel::EnableTimeExceptForMutex);
```

Comment:
Great! How about we make it a config? raftstore has a similar config: https://github.com/tikv/tikv/blob/master/components/raftstore/src/store/config.rs#L184

Reply:
@yiwu-arbug Yes! If the performance degrade is notable then it needs a config to opt-in/out.